pkg/plugin/util/testdata/exampleFile.txt

```diff
@@ -1 +1 @@
-exampleTargetexampleCodeexampleCodeexampleCodeexampleCodeexampleCodeexampleCodeexampleCode
\ No newline at end of file
+exampleTargetexampleCodeexampleCodeexampleCodeexampleCodeexampleCodeexampleCodeexampleCodeexampleCodeexampleCodeexampleCodeexampleCodeexampleCodeexampleCode
```

Comment:
@jonas-jonas 

You need rebase it with the master
And then, revert the changes here
Keep it sync with make generated,

Reply:
Done.